qutip/utilities.py

```diff
@@ -108,7 +108,7 @@ def clebsch(j1, j2, j3, m1, m2, m3):
     C = np.sqrt((2.0 * j3 + 1.0)*_to_long(c_factor))
 
     s_factors = np.zeros(((vmax + 1 - vmin), (int(j1 + j2 + j3))), np.int32)
-    sign = (-1) ** (vmin + j2 + m2)
+    sign = int((-1) ** (vmin + j2 + m2))
```

Comment:
Why did this need to be wrapped with `int`?

Reply:
`clebsch` is computed using large integer over 32 and sometime 64 bits.
If the sign is a 0d numpy array it overflow.

Reply:
Ah. Perhaps add a comment above this line with that explanation?